cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AuthorizationHandler.java

```diff
@@ -81,9 +81,15 @@ public void authorized(HttpRequest httpRequest, HttpResponder httpResponder) thr
       throw new BadRequestException("Missing request body");
     }
 
-    CheckAuthorizedResponse response = new CheckAuthorizedResponse(
-      auth.authorized(request.getEntity(), request.getUser(), request.getActions()));
-    httpResponder.sendJson(HttpResponseStatus.OK, response);
+    if (request.getActions() == null) {
```

Comment:
In fact, as part of [CDAP-5133](https://issues.cask.co/browse/CDAP-5133), I propose to remove this API altogether and instead have an API that returns all the privileges of a given user. But that will be done later.